sdk/python/kfp/dsl/_component.py

```diff
@@ -37,16 +37,16 @@ def python_component(name, description=None, base_image=None, target_component_f
   Returns:
       The same function (with some metadata fields set).
 
-  Usage:
-  ```python
-  @dsl.python_component(
-    name='my awesome component',
-    description='Come, Let's play',
-    base_image='tensorflow/tensorflow:1.11.0-py3',
-  )
-  def my_component(a: str, b: int) -> str:
-    ...
-  ```
+  Example:
+    ::
```

Comment:
Another common problem I fixed was setting injecting literal (`::`) so Sphinx would interpret the example as a formatted code block.

Looks like Sphinx needs the `::` to be followed by a blank line then the code block indented for it to actually format nicely. Guessing there is likely something we can do with a linter to begin to enforce this but for now I've just fixed them manually.